utility/src/main/java/org/oppia/util/parser/GlideImageLoader.kt

```diff
@@ -2,18 +2,22 @@ package org.oppia.util.parser
 
 import android.content.Context
 import android.graphics.Bitmap
+import android.graphics.drawable.PictureDrawable
 import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.request.RequestOptions
 import com.bumptech.glide.request.target.CustomTarget
-import javax.inject.Inject
 import org.oppia.util.caching.AssetRepository
 import org.oppia.util.caching.CacheAssetsLocally
+import javax.inject.Inject
```

Comment:
This should come above otherwise it will give lint issues.

Reply:
Done.